pyvisa_py/usb.py

```diff
@@ -336,7 +372,7 @@ def list_resources() -> List[str]:
                 serial = dev.serial_number
             except (NotImplementedError, ValueError, usb.USBError):
                 msg = (
-                    "Found a device whose serial number cannot be read."
+                    "Found a USB RAW device whose serial number cannot be read."
```

Comment:
can't we run into the same permissions issue here?

Reply:
USB RAW are weird because there are supported by NI but are not part of the IVI specs. If we can provide more information we should though.

Reply:
I observed in pyvisa issue [Linux host and getting permissions to talk to a USBTMC device](https://github.com/pyvisa/pyvisa/issues/609#issuecomment-925396821) 
USBTMC devices show up as both /dev/usbtmcN and /dev/bus/usb/NNN/NNN
Both instances need to have suitable permissions for pyvisa with pyvisa-py to be able to communicate

If only the /dev/usbtmcN has permissions then you get `WARNING Found a device whose serial number cannot be read`